Nuoqte | 2024/12/17 , 16:14:08 -> Adding job tentatively -- it will be properly scheduled when the scheduler starts
Nuoqte | 2024/12/17 , 16:14:08 -> Error loading banned users: (sqlite3.OperationalError) no such table: banned_accounts
[SQL: SELECT banned_accounts.user_id 
FROM banned_accounts]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
Nuoqte | 2024/12/17 , 16:14:08 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:14:08 -> Connecting...
Nuoqte | 2024/12/17 , 16:14:18 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:14:19 -> Connecting...
Nuoqte | 2024/12/17 , 16:14:29 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:14:30 -> Connecting...
Nuoqte | 2024/12/17 , 16:14:40 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:14:41 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:14:41 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:14:42 -> Disconnected
Nuoqte | 2024/12/17 , 16:14:42 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:14:42 -> Connecting...
Nuoqte | 2024/12/17 , 16:14:52 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:14:53 -> Connecting...
Nuoqte | 2024/12/17 , 16:15:03 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:15:04 -> Connecting...
Nuoqte | 2024/12/17 , 16:15:14 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:15:15 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:15:15 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:15:16 -> Disconnected
Nuoqte | 2024/12/17 , 16:15:16 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:15:16 -> Connecting...
Nuoqte | 2024/12/17 , 16:15:26 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:15:27 -> Connecting...
Nuoqte | 2024/12/17 , 16:15:37 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:15:38 -> Connecting...
Nuoqte | 2024/12/17 , 16:15:48 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:15:49 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:15:49 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:15:50 -> Disconnected
Nuoqte | 2024/12/17 , 16:15:50 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:15:50 -> Connecting...
Nuoqte | 2024/12/17 , 16:16:00 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:16:01 -> Connecting...
Nuoqte | 2024/12/17 , 16:16:11 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:16:12 -> Connecting...
Nuoqte | 2024/12/17 , 16:16:22 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:16:23 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:16:23 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:16:24 -> Disconnected
Nuoqte | 2024/12/17 , 16:16:24 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:16:24 -> Connecting...
Nuoqte | 2024/12/17 , 16:16:34 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:16:35 -> Connecting...
Nuoqte | 2024/12/17 , 16:18:36 -> Adding job tentatively -- it will be properly scheduled when the scheduler starts
Nuoqte | 2024/12/17 , 16:18:36 -> Error loading banned users: (sqlite3.OperationalError) no such table: banned_accounts
[SQL: SELECT banned_accounts.user_id 
FROM banned_accounts]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
Nuoqte | 2024/12/17 , 16:18:37 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:18:37 -> Connecting...
Nuoqte | 2024/12/17 , 16:19:07 -> Adding job tentatively -- it will be properly scheduled when the scheduler starts
Nuoqte | 2024/12/17 , 16:19:07 -> Error loading banned users: (sqlite3.OperationalError) no such table: banned_accounts
[SQL: SELECT banned_accounts.user_id 
FROM banned_accounts]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
Nuoqte | 2024/12/17 , 16:19:07 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:19:07 -> Connecting...
Nuoqte | 2024/12/17 , 16:19:17 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:19:18 -> Connecting...
Nuoqte | 2024/12/17 , 16:19:28 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:19:29 -> Connecting...
Nuoqte | 2024/12/17 , 16:20:29 -> Adding job tentatively -- it will be properly scheduled when the scheduler starts
Nuoqte | 2024/12/17 , 16:20:29 -> Error loading banned users: (sqlite3.OperationalError) no such table: banned_accounts
[SQL: SELECT banned_accounts.user_id 
FROM banned_accounts]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
Nuoqte | 2024/12/17 , 16:20:29 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:20:29 -> Connecting...
Nuoqte | 2024/12/17 , 16:20:39 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:20:40 -> Connecting...
Nuoqte | 2024/12/17 , 16:20:50 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:20:51 -> Connecting...
Nuoqte | 2024/12/17 , 16:21:01 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:21:02 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:21:02 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:21:03 -> Disconnected
Nuoqte | 2024/12/17 , 16:21:03 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:21:03 -> Connecting...
Nuoqte | 2024/12/17 , 16:21:13 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:21:14 -> Connecting...
Nuoqte | 2024/12/17 , 16:21:24 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:21:25 -> Connecting...
Nuoqte | 2024/12/17 , 16:21:35 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:21:36 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:21:36 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:21:37 -> Disconnected
Nuoqte | 2024/12/17 , 16:21:37 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:21:37 -> Connecting...
Nuoqte | 2024/12/17 , 16:21:47 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:21:48 -> Connecting...
Nuoqte | 2024/12/17 , 16:21:58 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:21:59 -> Connecting...
Nuoqte | 2024/12/17 , 16:22:09 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:22:10 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:22:10 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:22:11 -> Disconnected
Nuoqte | 2024/12/17 , 16:22:11 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:22:11 -> Connecting...
Nuoqte | 2024/12/17 , 16:22:21 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:22:22 -> Connecting...
Nuoqte | 2024/12/17 , 16:22:32 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:22:33 -> Connecting...
Nuoqte | 2024/12/17 , 16:22:57 -> Adding job tentatively -- it will be properly scheduled when the scheduler starts
Nuoqte | 2024/12/17 , 16:22:57 -> Error loading banned users: (sqlite3.OperationalError) no such table: banned_accounts
[SQL: SELECT banned_accounts.user_id 
FROM banned_accounts]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
Nuoqte | 2024/12/17 , 16:22:57 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:22:57 -> Connecting...
Nuoqte | 2024/12/17 , 16:23:07 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:23:08 -> Connecting...
Nuoqte | 2024/12/17 , 16:23:18 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:23:19 -> Connecting...
Nuoqte | 2024/12/17 , 16:23:29 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:23:30 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:23:30 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:23:31 -> Disconnected
Nuoqte | 2024/12/17 , 16:23:31 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:23:31 -> Connecting...
Nuoqte | 2024/12/17 , 16:23:41 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:23:42 -> Connecting...
Nuoqte | 2024/12/17 , 16:23:52 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:23:53 -> Connecting...
Nuoqte | 2024/12/17 , 16:26:25 -> Adding job tentatively -- it will be properly scheduled when the scheduler starts
Nuoqte | 2024/12/17 , 16:26:25 -> Error loading banned users: (sqlite3.OperationalError) no such table: banned_accounts
[SQL: SELECT banned_accounts.user_id 
FROM banned_accounts]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
Nuoqte | 2024/12/17 , 16:26:26 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:26:26 -> Connecting...
Nuoqte | 2024/12/17 , 16:26:36 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:26:37 -> Connecting...
Nuoqte | 2024/12/17 , 16:26:47 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:26:48 -> Connecting...
Nuoqte | 2024/12/17 , 16:26:58 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:26:59 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:26:59 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:27:00 -> Disconnected
Nuoqte | 2024/12/17 , 16:27:00 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:27:00 -> Connecting...
Nuoqte | 2024/12/17 , 16:27:10 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:27:11 -> Connecting...
Nuoqte | 2024/12/17 , 16:27:21 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:27:22 -> Connecting...
Nuoqte | 2024/12/17 , 16:27:32 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:27:33 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:27:33 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:27:34 -> Disconnected
Nuoqte | 2024/12/17 , 16:27:34 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:27:34 -> Connecting...
Nuoqte | 2024/12/17 , 16:27:44 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:27:45 -> Connecting...
Nuoqte | 2024/12/17 , 16:27:55 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:27:56 -> Connecting...
Nuoqte | 2024/12/17 , 16:28:06 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:28:07 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:28:07 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:28:08 -> Disconnected
Nuoqte | 2024/12/17 , 16:28:08 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:28:08 -> Connecting...
Nuoqte | 2024/12/17 , 16:28:18 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:28:19 -> Connecting...
Nuoqte | 2024/12/17 , 16:28:29 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:28:30 -> Connecting...
Nuoqte | 2024/12/17 , 16:28:40 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:28:41 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:28:41 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:28:42 -> Disconnected
Nuoqte | 2024/12/17 , 16:28:42 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:28:42 -> Connecting...
Nuoqte | 2024/12/17 , 16:28:52 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:28:53 -> Connecting...
Nuoqte | 2024/12/17 , 16:29:03 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:29:04 -> Connecting...
Nuoqte | 2024/12/17 , 16:29:14 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:29:15 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:29:15 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:29:16 -> Disconnected
Nuoqte | 2024/12/17 , 16:29:16 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:29:16 -> Connecting...
Nuoqte | 2024/12/17 , 16:29:26 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:29:27 -> Connecting...
Nuoqte | 2024/12/17 , 16:29:37 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:29:38 -> Connecting...
Nuoqte | 2024/12/17 , 16:29:48 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:29:49 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:29:49 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:29:49 -> Disconnected
Nuoqte | 2024/12/17 , 16:30:39 -> Adding job tentatively -- it will be properly scheduled when the scheduler starts
Nuoqte | 2024/12/17 , 16:30:39 -> Error loading banned users: (sqlite3.OperationalError) no such table: banned_accounts
[SQL: SELECT banned_accounts.user_id 
FROM banned_accounts]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
Nuoqte | 2024/12/17 , 16:30:39 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:30:39 -> Connecting...
Nuoqte | 2024/12/17 , 16:30:50 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:30:51 -> Connecting...
Nuoqte | 2024/12/17 , 16:31:01 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:31:02 -> Connecting...
Nuoqte | 2024/12/17 , 16:31:12 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:31:13 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:31:13 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:31:14 -> Disconnected
Nuoqte | 2024/12/17 , 16:31:14 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:31:14 -> Connecting...
Nuoqte | 2024/12/17 , 16:31:24 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:31:25 -> Connecting...
Nuoqte | 2024/12/17 , 16:31:35 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:31:36 -> Connecting...
Nuoqte | 2024/12/17 , 16:31:46 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:31:47 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:31:47 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:31:48 -> Disconnected
Nuoqte | 2024/12/17 , 16:31:48 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:31:48 -> Connecting...
Nuoqte | 2024/12/17 , 16:31:58 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:31:59 -> Connecting...
Nuoqte | 2024/12/17 , 16:32:09 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:32:10 -> Connecting...
Nuoqte | 2024/12/17 , 16:32:20 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:32:21 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:32:21 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:32:22 -> Disconnected
Nuoqte | 2024/12/17 , 16:32:22 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:32:22 -> Connecting...
Nuoqte | 2024/12/17 , 16:32:32 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:32:33 -> Connecting...
Nuoqte | 2024/12/17 , 16:32:43 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:32:44 -> Connecting...
Nuoqte | 2024/12/17 , 16:32:54 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:32:55 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:32:55 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:32:56 -> Disconnected
Nuoqte | 2024/12/17 , 16:32:56 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:32:56 -> Connecting...
Nuoqte | 2024/12/17 , 16:33:06 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:33:07 -> Connecting...
Nuoqte | 2024/12/17 , 16:33:17 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:33:18 -> Connecting...
Nuoqte | 2024/12/17 , 16:33:28 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:33:29 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:33:29 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:33:30 -> Disconnected
Nuoqte | 2024/12/17 , 16:33:30 -> Start creating a new auth key on DC2
Nuoqte | 2024/12/17 , 16:33:30 -> Connecting...
Nuoqte | 2024/12/17 , 16:33:40 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:33:41 -> Connecting...
Nuoqte | 2024/12/17 , 16:33:51 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:33:52 -> Connecting...
Nuoqte | 2024/12/17 , 16:34:02 -> Unable to connect due to network issues: Connection timed out
Nuoqte | 2024/12/17 , 16:34:03 -> Connection failed! Trying again...
Nuoqte | 2024/12/17 , 16:34:03 -> Retrying due to ConnectionError: 
Nuoqte | 2024/12/17 , 16:34:03 -> Disconnected
